app/models/conference_preference.rb

```diff
@@ -9,13 +8,11 @@ class ConferencePreference < ActiveRecord::Base
 
   private
 
-    def terms_accepted?
+    def terms
       conference = first_conference_id.present? || second_conference_id.present?
       terms_checkbox = terms_of_ticket == '1' && terms_of_travel == '1'
       first_case = conference.present? && terms_checkbox.present?
       second_case = conference.blank? && terms_checkbox.blank?
-      errors.add(:team, "If you choose any conference you must accept the terms") if conference && !terms_checkbox
-      errors.add(:team, "If you accept terms you must choose any conference") if !conference && terms_checkbox
       first_case || second_case
```

Comment:
Can you come up with better names for this?

Maybe the whole thing as something along the lines of
```ruby
def terms_checked?
  conference_chosen = first_conference_id || second_conference_id
  terms_checked = terms_of_ticket == '1' && terms_of_travel == '1'
  (conference_chosen && terms_checked) || (!conference_chosen && !terms_checked)
end
```